pkgs/by-name/ve/vesktop/package.nix

```diff
@@ -28,7 +29,7 @@ stdenv.mkDerivation (finalAttrs: {
     owner = "Vencord";
     repo = "Vesktop";
     rev = "v${finalAttrs.version}";
-    hash = "sha256-Ot2O5J1wUZAWgdpJNaEUSwtbcNqDdGhzuCtx8Qg+4gg=";
+    hash = "sha256-wGOyDGY0FpAVS5+MTiKrOpDyd13ng0RLGAENW5tXuR4=";
   };
```

Comment:
this is odd :/

Reply:
Indeed but it is required for it to work and for the patch to be applied 🤷‍♀️

Reply:
Where do you have this hash from? It seems like there is no version 0.4.4 anymore, at least there's no tag for it in the GitHub repo.

Reply:
> Where do you have this hash from? It seems like there is no version 0.4.4 anymore, at least there's no tag for it in the GitHub repo.

It's from 0.4.3. I have _no_ idea why 0.4.4 just poofed like that

Reply:
We cannot use the hash for 0.4.3 here. I see two options:

* rolling back to 0.4.3. I don’t know whether this is possible or if there’re db migrations or something like that
* updating to 1.5.0 (=backporting the unstable bump). I haven’t checked in detail yet if it’s applicable for backporting, but seems okay on first sight.

Reply:
> We cannot use the hash for 0.4.3 here. I see two options:
> 
> * rolling back to 0.4.3. I don’t know whether this is possible or if there’re db migrations or something like that
> * updating to 1.5.0 (=backporting the unstable bump). I haven’t checked in detail yet if it’s applicable for backporting, but seems okay on first sight.

Updating to 1.x is a non-option because it is a breaking change that forces people to re-login into their accounts, and had incompatible config formats. TBH I'm not entirely sure why the version is at 0.4.4 in the first place and why it seemingly just disappeared upstream — I'll look at the PRs opened before and check out why

Reply:
This was the bump 0.4.4 commit: https://github.com/Vencord/Vesktop/commit/7efa54687a3cb42ab662bc57c7b3ca1d951ace29

Reply:
By looking at forks, I found this tag: https://github.com/advantageous-overtake/Vesktop-Based/releases/tag/v0.4.4

So commit https://github.com/Vencord/Vesktop/commit/032b94e81da03b501fd69c46790fed9f2fac8090 was the commit 0.4.4 was tagged with

Reply:
Alright, I'm just gonna assume that commit is indeed 0.4.4 and try to recalculate the hash and see if it fits